cli/lib/util.js

```diff
@@ -286,14 +288,29 @@ const util = {
     .value()
   },
 
-  getOriginalNodeOptions (options) {
+  getOriginalNodeOptions ({ processVersions }) {
```

Comment:
Could you get `proccess.versions` here instead of passing it in? Is this called with the same node version that launched the cli?